run.sh

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env sh
+# Convenience script to run the wikiparser with the maps generator as a drop-in replacement for the scraper.
+#
+# 1. Finds latest map build.
```

Comment:
It would be great to clarify why the latest map build is needed at all.

Reply:
Am I correctly understanding the issue with the current approach?
1. Generator builds maps and creates csv/txt wiki ids files.
2. Wikiparser runs and generates articles.
3. Generator should be run again to reuse generated articles?..

Reply:
To replace the workflow with the scraper:
1. The generator needs to be run at least to the ["Features" stage](https://github.com/organicmaps/organicmaps/blob/a1b596bdc64ed5db3eabf6b1e331411aa2e4ab03/tools/python/maps_generator/generator/stages_declaration.py#L116) in order to generate the wiki files.
2. Running Wikiparser is a replacement for the ["DownloadDescriptions" stage](https://github.com/organicmaps/organicmaps/blob/a1b596bdc64ed5db3eabf6b1e331411aa2e4ab03/tools/python/maps_generator/generator/stages_declaration.py#L144)
3. The generator can now be run from the ["Descriptions" stage](https://github.com/organicmaps/organicmaps/blob/a1b596bdc64ed5db3eabf6b1e331411aa2e4ab03/tools/python/maps_generator/generator/stages_declaration.py#L292).

So if you'd like to do it in one run, we could update the generator to call wikiparser.
Or we could tweak the generator to only output the descriptions and continue so you can run wikiparser out-of-band.

Reply:
Updating the generator to call wikiparser means that generator should wait until wikiparser finishes, right? This approach may be a good temporary start, but we aim to speed up the map generation process as much as possible. That's why the ideal solution would (likely) be to start generator and wikiparser in parallel, as soon as a new osm planet dump is available (or maybe start wikiparser before the generator). So when the generator needs descriptions they will already be available.
That's why it was important to focus on speedy articles extraction/processing from the start.

 WDYT?

Reply:
Yes, it replaces the blocking scraper script in it's current form.
I thought it would be better to start with this working and then separate and speed up the process.
To separate fully from the generator we need to finish #19.